modules/shadertools/test/lib/transpile-shader-compress.js

```diff
@@ -0,0 +1,18 @@
+// copied from rollup-plugin-glslify
+export function compressShader(code) {
```

Comment:
Nits:
- `minifyShader`?
- Consistent names between main export and file name: `minify-shader.js`?
- I find it helpful to declare regexps as named well-named constants. This makes the code more self-documenting, it is quite hard to understand what a long regexp is intendeded to do.
- This could potentially be added as an export to the shadertools package (if so it needs a test case).

Reply:
1, 2 - updated
3 - right, this could be part of a larger overhaul
4- agree, when it's needed